src/transaction.js

```diff
@@ -363,6 +363,8 @@ const transaction = {
    *  'timestamp': 1,
    * }
    *
+   * @throws {ConstantNotSet} If the weight constants are not set yet
```

Comment:
This function does not throw `ConstantNotSet` but it uses one that does and we don't catch the exception. 

Should we have this docstring here? @jansegre your input is also appreciated.

Reply:
I think in this case where you are not catching the exception you should add in the docstring.